src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -162,7 +166,7 @@ public async Task<(Orchestrator, Recommendation, CloudApplication)> InitializeDe
 
                 // preset settings for deployment based on last deployment.
                 selectedRecommendation = await GetSelectedRecommendationFromPreviousDeployment(recommendations, deployedApplication, userDeploymentSettings);
-            } 
+            }

```

Comment:
there are several whitespace changes in this class. what is causing them?

Reply:
same as above.